Veil-light-wallet-iPhone-sideload.html

```diff
@@ -0,0 +1,117 @@
+        <ol>
+            <li>Open AltStore on your computer and ensure your iPhone is connected. Be sure to check if your phone is actually connected to <br>
+                AltStore on your computer.</li>
+            <li>After verifying connection. Open AltStore on your phone and Select My Apps, and click on the "+" icon in the left upper side of the menu.<br>
+                Look and find the Veil Light Wallet IPA file you downloaded.</li> 
+            <figure>
+                <img src="light-wallet-pictures/add_app_example.png" alt="add_app_example">
+                <figcaption>Press the plus sign icon.</figcaption>
+            </figure>    
+                <li>If you did not store the IPA in iCloud. Find the Light Wallet IPA and go to step 5.</li> 
+                <ul>If you stored your IPA in the iCloud. <br>
+                 Looks towards the bottom right. Find and press browse.</ul>
+            <figure>
+                <img src="light-wallet-pictures/browse_button_example.png" alt="browse_example">
+                <figcaption>Press the browse file icon.</figcaption>
+            </figure>   
+                <li>Then look for and press the iCloud Drive.</li> 
+            <figure>
+                <img src="light-wallet-pictures/iCloud_drive_example.png" alt="iCloud_example">
+                <figcaption>Press iCloud Drive file location.</figcaption>
+            </figure>   
+                <li>Then you should be able to see and select the IPA file. <br>
+                Select the IPA file.</li>
+            <figure>
+                <img src="light-wallet-pictures/IPA_example.png" alt="IPA_example">
+                <figcaption>IPA file should look like this.</figcaption>
+            </figure>   
+                <li>AltStore should begin to sideload the Veil Light Wallet onto your iPhone.</li>
+                <li>The Veil Light Wallet should now be in your My Apps tab and availble on your iPhone.</li>
+            <figure>
+                <img src="light-wallet-pictures/wallet_Myapps.png" alt="wallet_Myapps">
+                <figcaption>Light wallet added to My Apps.</figcaption>
+            </figure>   
+        </ol>
+
+    <h2>Launching Veil Light Wallet:</h2>
+        <ol>
+            <li>Press the Veil icon to open the Veil Light Wallet on your iPhone. It should be available on you homescreen.</li>
+            <li>Configure any initial settings, such as language.</li>
+            <li>Ensure you record and securely store your seed phrase generated during the setup. You can also enter a seed Phrase from a previous Veil wallet.</li>   
+            <li>After entering your wallet phrase correctly let your wallet load.</li> 
+            <li>The wallet should be ready to use in a few seconds up to a 1/2 hour.</li>
+            <li>Note that since Veil is private by default, this light wallet uses RingCT, and basecoin will not be usable, visible, or available with this light wallet.</li>
```

Comment:
basecoin and CT, only RingCT available for now (in future there probably will be option to automatically convert CT to RingCT, but now it's unavailable)

Reply:
Updated.